damage_window.py

```diff
@@ -0,0 +1,234 @@
+import cv2
+import numpy as np
+
+class Rectangle(object):
+    def __init__(self, x, y, w, h):
+        '''
+        x, y represents top left point
+        '''
+        self.x = x
+        self.y = y
+        self.w = w
+        self.h = h
+
+    def corners(self):
+        '''
+        returns top-left and bottom-right corners
+        '''
```

Comment:
Bit of a misleading function name

Reply:
returns corners of a rectangle. you got a suggestion